mono/mini/mini-ppc.h

```diff
@@ -323,4 +343,32 @@ void mono_ppc_patch_plt_entry (guint8 *code, gpointer *got, mgreg_t *regs, guint
 
 void mono_ppc_set_func_into_sigctx (void *sigctx, void *func);
 
+
+// Debugging macros for ELF ABI v2
+#define DEBUG_ELFABIV2
```

Comment:
I actually have used them outside of mini-ppc.c which is why they are in the header file.  Good catch on the #define which I missed.